Documentation/project-proposal.md

```diff
@@ -37,58 +37,151 @@ Stakeholders:
 
 ## User Stories
 
-### Feature 1. User Management
-
-- **User Story**: As a potential user, I want to register an account with Spendr to start tracking my expenses so that I can start keep track of my expenses.
-  - **Acceptance Criteria**:
-    - Can register with username and password.
-
-- **User Story**: As a registered user, I want to log in to my Spendr account to access my expenses and budgets so that I can add my expenses.
-  - **Acceptance Criteria**:
-    - Can securely log in using username and password.
-    - Can log out of the account.
-
-### Feature 2. Expense Entry and History
-
-- **User Story**: As a Spendr user, I want to quickly input my daily expenses so that I can view my history later.
-  - **Acceptance Criteria**:
-    - Can add an expense with category, amount, and date.
-    - The added expense is recorded and reflected immediately in my history.
-
-- **User Story**: As a Spendr user, I want to view and manage my expense history so that I have an easy way to understand my expenses.
-  - **Acceptance Criteria**:
-    - Can view a list of all recorded expenses.
-    - Can edit or delete an existing expense.
-
-### Feature 3. Budget Management
-
-- **User Story**: As a Spendr user, I want to set a monthly or weekly budget to control my spending so that I don't overspend.
-  - **Acceptance Criteria**:
-    - Can define a monthly or weekly budget amount.
-    - The budget is displayed prominently in the application.
-
-### Feature 4. Expense Analytics
-
-- **User Story**: As a Spendr user, I want visual insights on my spending habits so that I make informed financial decisions later.
-  - **Acceptance Criteria**:
-    - Can view graphical representations (like bar and pie charts) of my spending by category or time.
-    - Can see a breakdown of expenses for specific periods (weekly, monthly, yearly).
-
-### Feature 5. Group Expense Splitting
-
-- **User Story**: When I have shared expenses, I want to split them among friends or roommates so that I can keep track of who owes me money.
-  - **Acceptance Criteria**:
-    - Can add usernames when inputting an expense.
-    - Can split an expense equally or by percentages.
-    - The application calculates and displays the share each individual owes.
-    - The user that made the expense or the individual that owe, can mark as paid.
+### User Management
+
+1. As a new user, I want to create an account so that I can start using the platform's functionalities.\
+**Acceptance Criteria:**\
+Given I am a new user,\
+When I open the app and navigate to the registration page,\
+Then I should be able to enter a unique username with a password and successfully register.
+2. As a logged-out user, I want to be able to sign in to the website so that I can access my personal profile.\
+**Acceptance Criteria:**\
+Given I am a logged-out user,\
+When I open the app and prompted to log in,\
+Then I should be able to successfully log in with my unique username and password.
```

Comment:
Could you add spaces in between the list just so its easier to read?
```suggestion
1. As a new user, I want to create an account so that I can start using the platform's functionalities.\
**Acceptance Criteria:**\
Given I am a new user,\
When I open the app and navigate to the registration page,\
Then I should be able to enter a unique username with a password and successfully register.

2. As a logged-out user, I want to be able to sign in to the website so that I can access my personal profile.\
**Acceptance Criteria:**\
Given I am a logged-out user,\
When I open the app and prompted to log in,\
Then I should be able to successfully log in with my unique username and password.

```